app/config/services.php

```diff
@@ -116,3 +116,13 @@ function ($view) {
 
     return $session;
 });
+
+/**
+ * Register the workers' helpers
+ */
+$di->set(
```

Comment:
use $di->set(Class::class, Class::class) syntax to register new stuff to your DI